modules/exploits/multi/php/jorani_path_trav.rb

```diff
@@ -0,0 +1,164 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name' => 'Jorani unauthenticated Remote Code Execution',
+                      'Description' => '
+          This module exploits an unauthenticated Remote Code Execution in Jorani.
+          It abuses 3 vulnerability: log poisonning and redirection bypass via header spoofing, then it uses path traversal to trigger the vulnerability.
+          It has been tested on Jorani 1.0.0.
+        ',
+                      'License' => MSF_LICENSE,
+                      'Author' =>
+                        [
+                          'RIOUX Guilhem (jrjgjk)'
+                        ],
+                      'References' =>
+                        [
+                          %w[CVE CVE-2023-26469],
+                          ['URL', 'https://github.com/Orange-Cyberdefense/CVE-repository/blob/master/PoCs/CVE_Jorani.py']
```

Comment:
Link is dead :/

Reply:
Yep for the moment it is, I'm waiting for it to be publish at this URL (soon)

Reply:
Link is no longer dead and is active. Resolving this issue.